src/org/labkey/test/components/ext4/RadioButton.java

```
@@ -32,6 +33,16 @@ public static RadioButtonFinder RadioButton()
         return new RadioButtonFinder();
     }
 
+    @Override
+    public void set(@NotNull Boolean checked)
+    {
+        if (!checked)
```

seems this is an argument against radiobutton extending checkbox, but this is a good idea- fail fast and hard rather than silently